[IMP] Recruitment: adding new hiring flow doc (#5994)
odoo/documentation: content/applications/hr/recruitment.rst

Applications`. All applications for all job positions are presented in a list view. To view the
refused candidates only, click :guilabel:`Filters`, then click :guilabel:`Archived/Refused`. All
candidates that have been archived and/or refused appear in the list. To further organize this list
and view refusals by job position, click :guilabel:`≣ Group By`, then click :guilabel:`Job`.

.. note::
   Email templates in Odoo are capable of using dynamic content, which are customized pieces of data
   that populate dynamic placeholders. For example, the applicant's name can be a piece of dynamic
   content. The applicant's name will appear anytime that dynamic placeholder appears on the email
   template. For information on modifying email templates, refer to :doc:`this documentation <Job
   positions>`.

New applicant
~~~~~~~~~~~~~

If an applicant should be added and does not have an applicant card already (they have not
applied online for the job) an applicant card can be easily added from the *Recruitment*
application in one of two ways, using either the :guilabel:`Quick add` or :guilabel:`Create`

> buttons.

Author: Good catch! Glad we worked this out- I'll be sure this is in the job position setup doc since that's the best place for it. Thanks!

  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is
  visible on the individual card.
- :guilabel:`Applicant's Name`: Enter the applicant's name.
- :guilabel:`Email`: Enter the applicant's email address.
- :guilabel:`Phone`: Enter the applicant's phone number.
- :guilabel:`Mobile`: Enter the applicant's mobile number.
- :guilabel:`Degree`: Select the applicant's highest level of education earned from the drop-down
  menu. Options are :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates graduating the highest
  level of school before a Bachelor's degree, such as a high school or secondary school diploma,
  depending on the country.
- :guilabel:`Tags`: Select as many tags as desired from the drop-down menu. To add a tag that does
  not exist, type in the tag name, the click :guilabel:`Create "new tag"`.
- :guilabel:`Interviewer`: Select the employee who will be conducting the interview(s) from the
  drop-down menu. Anyone in the company can be selected as the interviewer, they do not have to be

>   a recruiter.

Author: Yes- I checked with Quentin. Because anyone can do the interview, and they don't have any other rights in recruitment (unlike the recruiter).

First interview
---------------

After a candidate has passed the :guilabel:`Initial Qualification` stage, they next move to the
:guilabel:`First Interview` stage. To move the applicant to the next stage, drag and drop the
applicant's card to the :guilabel:`First Interview` stage, or click on the :guilabel:`First
Interview` stage button at the top of the individual applicant's card.

When the candidate's card moves to the :guilabel:`First Interview` stage, an acknowledgement email
is automatically sent to the candidate. The email is created using an email template titled
:guilabel:`Applicant: Acknowledgement`. This email template is pre-configured in Odoo, and contains
dynamic placeholders for the job position, the recruiter's name, and the company's information. Once
the email is sent, the email appears in the chatter.

.. image:: recruitment/applicant-acknowledgement.png
   :align: center
   :alt: An email is automatically sent once an applicant is moved to the First Interview stage. The
         email is seen in the chatter, and is sent using a template.

Next, an activity is scheduled for the same day, titled :guilabel:`“Send mail for first interview”`.

> This planned activity appears in the chatter. The recruiter then sends an email to the applicant.

Author: I figured it out! When I was checking in runbot, I moved an app to this stage and i saw a planned activity- BUT it looks like it was set for the initial qualification stage (which is crazy) saying to schedule an interview in 2 weeks. Which I disagree with since you need to determine IF you want to give them an interview. But good catch- I tried several runbots before I figured out what I got wrong!

Email templates
~~~~~~~~~~~~~~~

To communicate with the applicant, Odoo has several pre-configured email templates that can be used.
The pre-configured email templates and when to use them are as follows:

- :guilabel:`Applicant: Acknowledgement`: This template is used to let the candidate know that their
  application was received. This email is automatically sent out once the candidate is in the

>   :guilabel:`Initial Qualification` stage.

Author: Good catch- I could have sworn I checked this on an empty 15 DB, and in runbot- but I must have crossed my wires! (Plus it doesn't make sense to me, it should be when we receive the application =)

   document.

To send an email using the :guilabel:`Applicant: Interest` email template, click :guilabel:`Send
message` in the chatter. A text box appears, as well as the candidate's email address. To use an
email template, click the :guilabel:`⤢ (Full composer)` icon in the bottom right corner of the
:guilabel:`Send Message` tab in the chatter.

.. image:: recruitment/full-composer.png
   :align: center
   :alt: Send an email using a template by clicking the full composer icon in the bottom right of
         the send message tab in the chatter.

An email pop-up loads, with the :guilabel:`Recipients` and :guilabel:`Subject` pre-populated. The
candidate's email address is entered on the :guilabel:`Recipients` line, and the :guilabel:`Subject`
is `Re: (Subject / Application Name)`. The email body is empty by default.

To use an email template, click the drop-down next to the :guilabel:`Load template` field in the
lower right section of the window. There are several pre-configured email templates to chose from.
Select the :guilabel:`Applicant: Interest` email template, and the template loads in the email body.

> This email template states that the applicant is shortlisted for either a phone call or an
> interview, and provides the contact information for the person who is listed as the
> :guilabel:`Interviewer` on the job position, explaining they will be contacted by that person.

Reviewer: ```suggestion

```
I think it would be best to keep this section generic to describe just the act of sending a message via a template, not a specific template.

Author: Good idea- they can read what each template says

If any attachments are to be added, click the :guilabel:`Attach A File` button in the lower left
corner. Navigate to the file to be attached, then click :guilabel:`Open` to attach it. To delete an
attachment, click the :guilabel:`X (Delete)` icon to the right of the attachment. If any changes
need to be made to the email, edit the email. If the edits should be saved to be used in the future,
the email can be saved as a new template. Click the :guilabel:`Save As New Template` button in the
bottom right corner. To send the email, click :guilabel:`Send` and the email will be sent to the
candidate. The email then appears in the chatter.

.. _recruitment/schedule:

Schedule a meeting
~~~~~~~~~~~~~~~~~~

When a date and time have been agreed upon for both the candidate and interviewer, the interview can
be scheduled. To schedule an interview, whether a phone or in-person interview, click the

> :guilabel:`Meeting` smart button at the top of the applicant's card.

Reviewer: ```suggestion
:guilabel:`Meeting` smart button at the top of the applicant's record.
```
to me, card implies we are still looking at the pipeline/kanban view.

  to see the meeting, :guilabel:`Private` allows only the attendees listed on the meeting to see the
  meeting, and :guilabel:`Only internal users` allows everyone in the company to see the meeting.
- :guilabel:`Show as`: Select from the drop-down whether the meeting time should appear as
  :guilabel:`Busy` or :guilabel:`Available` in the attendees' schedules. Setting this field to
  :guilabel:`Available` allows other meetings to be scheduled for the same time. It is recommended
  to set this field to :guilabel:`Busy`, so no other meetings can be scheduled for the attendees at
  the same time.

.. image:: recruitment/new-event.png
   :align: center
   :alt: A new meeting card with all the details populated, and ready to save.

Send meeting to attendees
*************************

Once changes have been entered and the meeting details are correct, the meeting can be sent to the
attendees via email or text message.

To send the meeting via email, click the :guilabel:`Email` button next to the list of attendees. A
:guilabel:`Contact Attendees` pop-up appears. A pre-formatted email using the default

> :guilabel:`Calendar: Event Update` email template populates the email body field. The
> :guilabel:`Recipients` are by default the followers of the document (job application) as well as
> the :guilabel:`Recruiter`. Add the applicant's email address to the list to send the email to the

Reviewer: ```suggestion
:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of the document (job application), as well as the :guilabel:`Recruiter` are added as :guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email to the
```

applicant as well. Make any other desired changes to the email. If an attachment is needed, click
the :guilabel:`Attach A File` button, navigate to the file, then click :guilabel:`Open`. Once the
email is ready to be sent, click :guilabel:`Send`.

.. image:: recruitment/email-event.png
   :align: center
   :alt: Enter the information to send the event via email.

To send the meeting via text message, click the :guilabel:`SMS` button next to the list of
attendees. A :guilabel:`Send SMS Text Message` pop-up appears. At the top, in a blue box,
information appears regarding the message. The first line lists how many records (phone numbers) are

> selected. It is *not* reccommended to check the box to send to all the records. The second line lists

Reviewer: Do we have a concrete reason why we don't want to send to all records? (Beyond the obvious spamming?) If so, it may be worth a "Warning" call out.

Author: Good idea!

  :guilabel:`Job Position` card. To modify the name, click the :guilabel:`External link` icon to the
  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
- :guilabel:`Department`: The department the job position falls under. To modify the department,
  click the :guilabel:`External link` icon to the right of the drop-down arrow. Make any changes,
  then click :guilabel:`Save`.
- :guilabel:`Force New Cars List`: Check this box to offer a new vehicle to the candidate.
- :guilabel:`Default Vehicle`: Using the drop-down menu, select the vehicle that will be assigned
  to the applicant.
- :guilabel:`Canteen Cost`: Enter the amount that is offered to the applicant for the cafeteria.
- :guilabel:`Contract Type`: Using the drop-down menu, select the type of contract that is being
  offered.

.. image:: recruitment/generate-link.png
   :align: center
   :alt: Format the offer to send to the candidate. Make any modifications before sending.

Send offer
~~~~~~~~~~

To send the offer to the applicant, click :guilabel:`Send`. An email pop-up appears, using the
:guilabel:`Applicant: Your Salary Package` email template. The :guilabel:`Recipients`,

> :guilabel:`Subject`, and email bdy are pre-populated based on the email template. If any attachments

Reviewer: ```suggestion
:guilabel:`Subject`, and email body are pre-populated based on the email template. If any attachments
```

==============
Applicant flow
==============

Odoo keeps all job applicants organized with a preconfigured series of steps, with specific stages
that each applicant goes through. Each stage has a specific step or steps that should be performed,
from scheduling a phone call, conducting an interview, or sending a job offer, to name a few. This
process is referred to as the 'applicant flow'.

When an applicant applies for a job position, an *applicant card* is created in Odoo's *Recruitment*
app for that specific job position. As the applicant progresses through the recruitment pipeline,
their card is moved from one stage to the next.

:ref:`Stages can be configured <modify>` so that an email is automatically sent out using

> a pre-selected templates soon as an applicant's card enters a stage. These automated emails are

Reviewer: ```suggestion
a pre-selected template as soon as an applicant's card enters a stage. These automated emails are
```

- :guilabel:`Yellow`: There are applicants in the stage with an activity either scheduled for today
  or due to be scheduled today.
- :guilabel:`Red`: There are applicants in the stage with overdue activities which need to be
  scheduled.
- :guilabel:`Gray`: There are either no activities currently scheduled and no future activities that
  are required to be scheduled, or there are no applicants currently in the stage.

.. image:: recruitment/stages.png
   :align: center
   :alt: Expand a folded column by clicking on it in the kanban view.

Customize stages
================

Stages can be modified, added, or deleted to best meet the needs of the hiring flow of the business.

New stage
---------

To create a new stage, click on :guilabel:`+ Add a Column` and a new column appears. Enter the title
for the new stage in the :guilabel:`Column title` field, then click :guilabel:`Add`. The new column
appears, and another new stage is available to create. If no new stages are needed, click anywhere
on the screen to close the new column.

.. image:: recruitment/add-column.png
   :align: center
   :alt: Add a column by clicking on Add Column.

.. _modify:

Modify stage
------------

> To modify the settings of a  stage, hover over the name of the stage, and a :guilabel:`⚙️ (gear)`

Reviewer: ```suggestion
To modify the settings of a stage, hover over the name of the stage, and a :guilabel:`⚙️ (gear)`
```

  position. This informs the applicant they can configure their salary package.

.. note::
   Email templates can be created, modified, and deleted to suit the needs of a business. For more
   information on email templates, refer to the :doc:`../general/email_communication/email_template`
   document.

To send an email using an email template, click :guilabel:`Send message` in the chatter. A text box
appears, as well as the applicant's email address. Click the :guilabel:`⤢ (Full composer)` icon in
the bottom right corner of the :guilabel:`Send Message` tab in the chatter.

.. image:: recruitment/full-composer.png
   :align: center
   :alt: Send an email using a template by clicking the full composer icon in the bottom right of
         the send message tab in the chatter.

An email pop-up loads, with the :guilabel:`Recipients` and :guilabel:`Subject` pre-populated. The
applicant's email address is entered on the :guilabel:`Recipients` line, and the :guilabel:`Subject`
is `Re: (Subject / Application Name)`. The email body is empty by default.

To use an email template, click the drop-down next to the :guilabel:`Load template` field in the

> lower right section of the window. There are several pre-configured email templates to chose from.

Reviewer: ```suggestion
lower right section of the window. There are several pre-configured email templates to choose from.
```

> lower right section of the window. There are several pre-configured email templates to chose from.

Reviewer: ```suggestion
lower right section of the window. There are several pre-configured email templates to choose from.
```
Same with this one, it says resolved but is still showing up for me as "chose" not "choose".

needed to the form.

New event card
**************

The fields available to populate or modify on the detailed :guilabel:`New Event` card are as
follows. The only required fields to enter are the :guilabel:`Meeting Subject`, :guilabel:`Starting
At`, and :guilabel:`Ending At` fields. Once the card details are entered, click :guilabel:`Save` to
save the changes and create the meeting.

- :guilabel:`Meeting Subject`: Enter the subject for the meeting. This should clearly indicate the
  purpose of the meeting. The default subject is the :guilabel:`Subject/Application Name` on the
  applicant's card.
- :guilabel:`Attendees`: Select the people who should be in attendance. The default employee listed
  is the person who is creating the meeting. Add as many other people as desired.

Meeting details tab
^^^^^^^^^^^^^^^^^^^

- :guilabel:`Starting At`: Using the drop-down calendar module, select the date and time for the
  meeting start time.
- :guilabel:`Ending At`: Using the drop-down calendar module, select the date and time for the
  meeting end time.
- :guilabel:`Duration`: This field auto populates based on the :guilabel:`Starting At` and

>   :guilabel:`Ending At` times entered. If the meeting time is adjusted, tis field automatically

Reviewer: ```suggestion
  :guilabel:`Ending At` times entered. If the meeting time is adjusted, this field automatically
```

>   :guilabel:`Ending At` times entered. If the meeting time is adjusted, tis field automatically

Reviewer: ```suggestion
  :guilabel:`Ending At` times entered. If the meeting time is adjusted, this field automatically
```
I know it says marked as resolved, but it looks like it is still "tis" and not "this"? I can't tell if the issue is on my end and how it is showing up for me, but may want to take a look? (Such a small issue, I'm sorry)

==============
Applicant flow
==============

Odoo keeps all job applicants organized with a preconfigured series of steps, with specific stages

> that each applicant goes through. Each stage has a specific step or steps that should be performed,

Author: Agree with the text, not the comma.

from scheduling a phone call, conducting an interview, or sending a job offer, to name a few. This
process is referred to as the 'applicant flow'.

When an applicant applies for a job position, an *applicant card* is created in Odoo's *Recruitment*
app for that specific job position. As the applicant progresses through the recruitment pipeline,

> their card is moved from one stage to the next.

Author: The card is created automatically, not the stage changes. I'll edit to make that clear.

:ref:`Stages can be configured <modify>` so that an email is automatically sent out using

> a pre-selected template as soon as an applicant's card enters a stage. These automated emails are

Reviewer 2: The template be accessed by going to configuration--> Job Positions --> select the position--> recruitment tab--> interview form??

> a pre-selected template as soon as an applicant's card enters a stage. These automated emails are

Author: Yes- you can- but I also changed this so that it links to the section that explains how to modify all of this, so I think we are good.

> a pre-selected template as soon as an applicant's card enters a stage. These automated emails are

Author: Also, the other doc in review is all abut the creation of an interview form in the new job position doc, which goes into detail for this, so I am not sure we need to add this.

defined on each stage in the hiring flow.

.. note::

>    The stages in the hiring flow are universal and apply to all job positions unless specified on

Author: I made up both terms (or thought I did!) and was interchanging them, even though I realized I should pick one. I changed everything to Applicant Flow, and only use the term hiring for a more general term where applicable.

>    the :guilabel:`Edit Column` form for a particular stage. The hiring flow with all its stages are

Author: I added a link to the section where this is done. I wanted to explain in general how the stages worked before getting into the nuts and bolts below.

   visible on all job positions.

Kanban view
===========

To access the kanban view for a job position, navigate to the main :menuselection:`Recruitment app
dashboard`. All job positions appear on the main dashboard. Click the :guilabel:`New Applications`
smart button on a job position card to navigate to the kanban view for all the applicants for that
particular job position.

.. image:: recruitment/new-applicants-button.png
   :align: center
   :alt: Main dashboard view of job position card, showing new applications button.

The kanban stages appear, with all the applicants populated in their respective columns, indicating
what stage they are currently in. There are five default stages in Odoo:

- :guilabel:`Initial Qualification`
- :guilabel:`First Interview`
- :guilabel:`Second Interview`
- :guilabel:`Contract Proposal`
- :guilabel:`Contract Signed`

The last column, :guilabel:`Contract Signed`, is folded by default, meaning that the column and the
applicants in it are hidden from view. To expand the folded stage and view the applicant cards for
that column, click anywhere on the thin gray column that says the stage name and the column expands,
revealing the applicants.

> Each stage has a color-coded bar beneath the stage name, giving information about the applicant's in

Author: changed it to status, since it is related to if an activity is late, etc.

icon appears. Click on the :guilabel:`⚙️ (gear)` icon and a menu appears. Click on the
:guilabel:`Edit Stage` option. An :guilabel:`Edit Column` form appears. Make any modifications to
the form, then click :guilabel:`Save` when done.

Edit column form
~~~~~~~~~~~~~~~~

The :guilabel:`Edit Column` form is where the stage's settings are configured. The only required
fields are the :guilabel:`Stage Name`, and the :guilabel:`Tooltips` section.

The fields to be populated or modified are:

- :guilabel:`Stage Name`: Type in a name for the stage.
- :guilabel:`Email Template`: Select an email template to be used from the drop-down menu. If a
  template is selected, when the applicant card enters the stage, an email will automatically be
  sent to the applicant using the selected template.
- :guilabel:`Folded in Kanban`: Check the box to have the stage appear folded (hidden) at all times
  in the default view.
- :guilabel:`Hired Stage`: Check the box if this stage indicates that the applicant is hired. When
  an applicant's card enters this stage, the card displays a :guilabel:`Hired` banner in the corner.

>   If this box is checked, this stage is used to determine the hire date of an applicant.

Author: Yup= Jess and I had to tinker a lot to find that out (I'll be honest, it was 99% Jess!) because you can set MULTIPLE STAGES as the hired stage! Which is crazy- and can't be fixed because the kanban is linked to other apps, so making a rule for that is not possible according to the devs (too many potential problems). So, we learned that the first stage that is set as a hiring stage is the date the applicant is hired (on their employee record).  After hashing it out with Jess, I decided not to mention that you can set multiple stages as the hired stage because that's crazy, unhelpful, and will likely confuse people (since instinctively, there should only be one hired stage). Plus if I mentioned it, I think it makes Odoo look bad, and thought best not to include that.

- :guilabel:`Job Specific`: If the stage only applies to a specific job position(s), select the job
  position(s) from the drop-down menu. Multiple job positions can be selected.

> - :guilabel:`Show in Referrals`: Check the box if this stage should be seen in the *Referrals*

Author: whoops! It appears after referrals is checked- thanks!

  application, and allow the referrer to accrue points when a referral of theirs reaches this stage.
- :guilabel:`Tooltips` section: There are three pre-configured labels (colored circles) for each
  applicant's card, indicating its status. The *names* for the label can be modified, but the label
  itself (the color) cannot. The default names and labels are: :guilabel:`In Progress` (gray),
  :guilabel:`Blocked` (red), and :guilabel:`Ready for Next Stage` (green).
- :guilabel:`Requirements`: Enter any internal notes for this stage explaining any requirements of
  the stage.

Delete stage
------------

If a stage is no longer needed, the stage can be deleted. To delete a stage, hover over the name of
the stage, and a :guilabel:`⚙️ (gear)` icon appears. Click on the :guilabel:`⚙️ (gear)` icon and a
drop-down menu is visible. Click :guilabel:`Delete` in the drop-down menu, and a pop-up warning
appears, asking :guilabel:`Are you sure you want to remove this column?` Click :guilabel:`OK` to
delete.

.. image:: recruitment/delete-stage.png
   :align: center

>    :alt: Delete a stage by accessing the drop-down menu from the gear icon.

Author: After reviewing- I agree it's confusing but doesn't really add anything impactful so I deleted it entirely.

>    :alt: Delete a stage by accessing the drop-down menu from the gear icon.

Author: I agree with it being in a better place so I made a new image for the Modify Stage section instead.

.. Important::
   If there are applicants currently in the stage being deleted, an error pops up when attempting to
   delete the stage, explaining that there are records in the stage which need to be deleted,
   archived, or moved to a different stage before deleting.

Email templates
===============

To communicate with the applicant, Odoo has several pre-configured email templates that can be used.

> The pre-configured email templates and when to use them are as follows:

Author: I added this second info, since the other info is in a detailed doc about adding a new job. I was having a hard time after the fleet doc deciding which info should be in which doc to prevent doubling up of info.

- :guilabel:`Applicant: Acknowledgement`: This template is used to let the applicant know that their
  application was received. This email is automatically sent out once the applicant is in the
  :guilabel:`First Interview` stage.
- :guilabel:`Applicant: Interest`: This template is used to let the applicant know that they have
  passed the :guilabel:`Initial Qualification` stage and they will be contacted to set up an
  interview with the recruiter.
- :guilabel:`Applicant: Not interested anymore`: This template is used when an applicant
  communicates that they are no longer interested in the position, and thanks them for their time
  and consideration.
- :guilabel:`Applicant: Refuse`: This template is used when an applicant is no longer being
  considered for the position.
- :guilabel:`Applicant: Your Salary Package`: This template is used when offering an applicant a
  position. This informs the applicant they can configure their salary package.

.. note::
   Email templates can be created, modified, and deleted to suit the needs of a business. For more
   information on email templates, refer to the :doc:`../general/email_communication/email_template`
   document.

> To send an email using an email template, click :guilabel:`Send message` in the chatter. A text box

Author: good specification!

appears, as well as the applicant's email address. Click the :guilabel:`⤢ (Full composer)` icon in
the bottom right corner of the :guilabel:`Send Message` tab in the chatter.

.. image:: recruitment/full-composer.png
   :align: center
   :alt: Send an email using a template by clicking the full composer icon in the bottom right of
         the send message tab in the chatter.

An email pop-up loads, with the :guilabel:`Recipients` and :guilabel:`Subject` pre-populated. The
applicant's email address is entered on the :guilabel:`Recipients` line, and the :guilabel:`Subject`
is `Re: (Subject / Application Name)`. The email body is empty by default.

> To use an email template, click the drop-down next to the :guilabel:`Load template` field in the

Author: I see in 150 it's use but in 15.2 it's load. I always use the most current version (and I think it is understood to the user as well).

dragged and dropped to the desired stage, or the stage can be modified on the applicant's card.

To change the stage on the applicant's card, click on the applicant's card in the kanban view to go
to a detailed view of the card. The current stage for the card is highlighted in purple at the top,
above the card. Simply click the desired stage for the card, and the stage changes. A note
indicating the stage change appears in the chatter. Navigate back to the kanban view by clicking on
:guilabel:`Applications` in the breadcrumb menu in the top left, and the applicant's card now
appears in the selected stage.

.. image:: recruitment/stage-change.png
   :align: center
   :alt: Change the stage of an applicant by clicking on the desired stage at the top of the
         applicant's card.

.. note::
   The following information is based on Odoo's default recruitment pipeline flow. Be advised that
   if modifications are made to create a customized recruitment pipeline flow, it will vary from the
   following information.

Initial qualification
---------------------

All applicants appear in the :guilabel:`Initial Qualification` stage after applying for a job

> position online. When the applicant's card is created, Odoo automatically populates the applicant's

Author: I double checked and clarified- it's both. if they did it online R if you click New and manually enter a new applicant- they are all placed there.

:guilabel:`Name`, :guilabel:`Email address`, and :guilabel:`Phone number` on the applicant's card.
This information is required when applying for a job position.

If the applicant entered any information in the :guilabel:`Short Introduction` section of the online
application, it is populated in the :guilabel:`Application Summary` tab at the bottom of the
applicant's card. If a resumé was attached to the online application, the resumé appears in the

> chatter, and is also stored in the *Documents* application. The resumé is accessible through both

Author: Yeah- you can see the thumbnail of it. I'll specify the attachments section.

the *Recruitment* and *Documents* applications.

Send interview
~~~~~~~~~~~~~~

At any point in the hiring process, an *interview* can be sent to the applicant to obtain more
information. These interviews are custom-made, and can be formatted in a variety of ways.

Odoo uses the term *interview*, but these can be thought of as questionnaires, surveys, tests,
certifications, etc. Custom interviews can be formatted to suit each individual job position's
needs. For example, a job position for a computer programmer could have an interview in the form of
a programming quiz to determine the skill level of the applicant. A job position for a restaurant
server could have a questionnaire inquiring about the applicant's availability, if the desired

> applicant needs to be available on weekend evenings. For more information on creating and editing

Author: Yes- this is the Job Positions document I was mentioning earlier. In this PR I have a note in here (near the top) stating that this document must be published after the job positions one. Currently, it does not exist to link to which is why it does not render, but once it is published, it will (and I will test it again, to make sure! =) 

This was my solution to the issue I had and you found in my fleet docs- where things were doubled up because you were asked to do the same thing in a different section, etc.

interview forms, refer to :doc:`this documentation <Job positions>`.

To send an interview to an applicant, click on an applicant's card to navigate to a detailed view of

> the applicant's card. At the top of the applicant's card, click the smart button labeled

Author: Oh ok, I thought they were all smart at the top of the page. Thanks!

added as a contact, add their email to the :guilabel:`Additional emails` line.

If any attachments need to be added, click the :guilabel:`Attachments` button and a file explorer
window appears. Navigate to the desired file, then click :guilabel:`Open` to attach it to the email.
The attachment loads, and is listed above the :guilabel:`Attachments` button.

If the emailed interview must be completed by a specific date, select a date for the
:guilabel:`Answer deadline`. Click on the line next to :guilabel:`Answer deadline`, and a calendar
module appears. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrows on either side of the
month to navigate to the desired month, then click on the :guilabel:`day` to select the date.

The :guilabel:`Mail template` field is pre-populated based on the configuration for the interview. A
different template can be chosen from the drop-down menu if desired. If a new template is selected,
the new email template loads in the email body.

To send the email with the interview link to the applicant, click :guilabel:`Send` at the bottom of
the email pop-up.

.. image:: recruitment/send-survey.png
   :align: center

>    :alt: Send a custom survey to an applicant using pre-configured templates.

Author: I didn't think one field was more important than the next since they were all explained (or there wasn't one thing to call out more than others)

> 

Author: Sorry- Odoo refers to them as Surveys, but they are the Interview Form (I dislike the terms they use for these!). But the "survey" is the interview form. I will specify that. As like the other interview forms, there's a link to the doc (not published yet) with all the specifics on how to do that.

.. note::
   In order to send an email to an applicant, there must be an email address on the applicant's
   card. If an email address is not entered on the applicant's card, when :guilabel:`Send Interview`
   is clicked, the applicant's card appears instead of the :guilabel:`Survey` pop-up window. Enter
   the email address in the email field, then click :guilabel:`Save`. Once the applicant's card is

>    saved, the applicant's card closes and the :guilabel:`Survey` pop-up window appears.

Author: I moved this note up- not to the very top but close.

Refuse
~~~~~~

At any point in the recruitment pipeline, an applicant can be refused. To refuse an applicant,
click on an applicant's card to navigate to a detailed view of the applicant's card. At the top of
the applicant's card, there are several smart buttons. Click the one labeled :guilabel:`Refuse`.

A :guilabel:`Refuse Reason` pop-up appears. Select the :guilabel:`Refuse Reason` from the drop-down
menu. The default refuse reasons in Odoo are :guilabel:`Doesn't fit the job requirements`,
:guilabel:`The applicant is not interested anymore`, and :guilabel:`The applicant gets a better
offer`. These three refusal reasons have pre-configured email templates associated with them.

> Additional refusal reasons can be created and existing ones can be modified or deleted. For more

Author: I added  an explanation on how to do this, and I added a link to the email templates block. Also, in the note at the end of this section I updated the links from the Job Positions doc to the email templates one- I think that's better!

information on configuring various settings in the *Recruitment* app, refer to :doc:`this

> documentation <Job positions>`.

Author: Same doc that is currently being reviewed/published. This will have to be published after that one, so the link works.

> documentation <Job positions>`.

Author: Yup- this will be till Job Positions is published (it is in final review now I believe). This doc must be published after.

An email template populates the :guilabel:`Email Template` field after a refusal reason is selected.
If an email should not be sent to the applicant, uncheck the :guilabel:`Send Email` checkbox. To
view the email template, click the :guilabel:`External Link` icon next to the email template
selected. The email template loads in the pop-up window, and can be modified if needed. After making
modifications, click :guilabel:`Save`. To close the email template, click :guilabel:`Discard`.

To send the refusal email to the applicant, click :guilabel:`Send`. The refusal email is sent to the
applicant, and a :guilabel:`Refused` banner appears on the applicant's card.

.. image:: recruitment/refuse.png
   :align: center
   :alt: Send an email explaining the reason why an applicant did not get the job, using

>          pre-configured refusal reasons and corresponding email templates.

Author: ooh I didn't think of that! Nice idea.

After refusal, the applicant's card is no longer visible in the job position's kanban view. To view
refused applicants, navigate to the :menuselection:`Recruitment app --> Applications --> All
Applications`. All applications for all job positions are presented in a list view. To view the
refused applicants only, click :guilabel:`Filters`, then click :guilabel:`Archived/Refused`. All
applicants that have been archived and/or refused appear in the list. To further organize this list
and view refusals by job position, click :guilabel:`≣ Group By`, then click :guilabel:`Job`.

.. note::
   Email templates in Odoo can use dynamic placeholders, which are customized pieces of data that
   populate dynamic content. For example, the applicant's name can be a piece of dynamic content.
   The applicant's name will appear anytime that dynamic placeholder appears on the email template.
   For information on modifying email templates, refer to :doc:`this documentation <Job positions>`.

New applicant
~~~~~~~~~~~~~

If an applicant should be added and does not have an applicant card already (they have not
applied online for the job) an applicant card can be easily added from the *Recruitment*

> application in one of two ways, using either the :guilabel:`Quick add` or :guilabel:`Create`

Author: Cleared it up and added links to anchors for the two buttons to explain.

buttons.

Quick add
*********

Quickly add a new applicant with the minimum information required using the :guilabel:`Quick add`
button. If not already in the kanban view for the job position to add an applicant to, navigate to
the main recruitment dashboard, and click the :guilabel:`New Applicants` smart button on the job
position card that the applicant should be added to. Then, click on the small :guilabel:`+ (Quick
add)` icon in the top right of the :guilabel:`Initial Qualification` stage to quickly add a new
applicant.

Enter the following information on the card:

- :guilabel:`Subject/Application Name`: This is the only required field. Enter the title for the
  card. Typically this is the applicant's name and job position being applied to, for example:
  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is

>   visible on the individual card.

Author: only if the name field is blank- I added this info

- :guilabel:`Applicant's Name`: Enter the applicant's name.
- :guilabel:`Email`: Enter the applicant's email address.
- :guilabel:`Applied Job`: The current job position populates this field. If needed, the job
  position can be changed by selecting a different position from the drop-down menu. If a different
  job position is selected, after the card is created, the card will appear in the selected job
  position.

After the information is entered, click :guilabel:`Add`. The applicant appears in the list, and a
new blank applicant card appears. Click either the :guilabel:`🗑️ (delete)` icon or click anywhere

> on the screen to close the card.

Author: Adde a sentence for that.

.. image:: recruitment/quick-add.png
   :align: center
   :alt: All the fields for a new applicant form entered when using the Quick Add option.

Create
******

Add a new applicant including all the relevant information using the :guilabel:`Create` button. If
not already in the kanban view for the job position to add an applicant to, navigate to the main
recruitment dashboard, and click the :guilabel:`New Applicants` smart button on the job position
card that the applicant should be added to. Then, click the :guilabel:`Create` smart button in the
top left of the kanban view and an applicant form loads.

> 

Author: true- but only after you enter the name (you can't click edit without a name for some reason, weird!).

Certain fields on the applicant card may be pre-populated, depending on how the job position is
configured. Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter` field, are

> all pre-populated, but this may vary.

Author: I'll remove the 'this can vary' since that's implied with the 'certain fields' comment.

Enter the following information on the new applicant form. Note that not all fields listed below may
be visible. Depending on installed applications and configurations, some fields may not be
displayed.

Applicant section
^^^^^^^^^^^^^^^^^

- :guilabel:`Subject/Application Name`: This is the only required field. Enter the title for the
  card. Typically this is the applicant's name and job position being applied to, for example:
  `Laura Smith - Experienced Developer`. This field is not visible in the kanban view, but is
  visible on the individual card.
- :guilabel:`Applicant's Name`: Enter the applicant's name.

> - :guilabel:`Email`: Enter the applicant's email address.

Reviewer 2: CC Email also appears in dev mode
![2023-10-23_14-06](https://github.com/odoo/documentation/assets/91896312/4f43f5be-dfe5-47fd-9289-3a2cc5c1f3de)
![2023-10-23_11-59](https://github.com/odoo/documentation/assets/91896312/5e6367d6-ab05-4851-a841-90b3a048eaa9)

Author: Noted and added that- I typically only put in directions for the default, I am never in dev mode, so I didn't think to mention or look for that. Do we typically add all the exclusions for Dev mode in our docs? I have not done so yet, unless the user must be in dev mode to do something - otherwise, I don't mention it.

- :guilabel:`Phone`: Enter the applicant's phone number.
- :guilabel:`Mobile`: Enter the applicant's mobile number.
- :guilabel:`Degree`: Select the applicant's highest level of education earned from the drop-down
  menu. Options are :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates graduating the highest
  level of school before a Bachelor's degree, such as a high school or secondary school diploma,
  depending on the country.
- :guilabel:`Tags`: Select as many tags as desired from the drop-down menu. To add a tag that does

>   not exist, type in the tag name, the click :guilabel:`Create "new tag"`.

Author: The name entered for the new tag appears in the quotes- I think it's understood, especially if they follow along, they will see it that way.   ![image](https://github.com/odoo/documentation/assets/121518652/1e9964a8-7637-432b-a0c2-874afe9a85e2)

- :guilabel:`Interviewer`: Select the employee who will be conducting the interview(s) from the

>   drop-down menu. Anyone in the company can be selected as the interviewer, they do not have to be

Reviewer 2: I don't show interviewer in 15.0.... where should I look?

Author: This is on the right side of the form- I checked, it appears in 15.0 and 15.2
![image](https://github.com/odoo/documentation/assets/121518652/9f5ec94c-e330-4690-ac4b-7c467a155304)

>   drop-down menu. Anyone in the company can be selected as the interviewer, they do not have to be

Author: Updated this with the following comment on recruiter

  a recruiter.

> - :guilabel:`Recruiter`: Select the person responsible for the entire recruitment process for the

Author: I dug in more- and I updated this. Based on permissions- which I assumed but was told otherwise- and did a second test, with permissions- to confirm how it works.

  job position.
- :guilabel:`Appreciation`: Click on one of the stars to select a rating for the applicant. One star
  indicates :guilabel:`Good`, two stars indicates :guilabel:`Very Good`, and three stars indicates
  :guilabel:`Excellent.`
- :guilabel:`Referred By User`: If referral points are to be earned for this job position in the
  *Referrals* application, select the employee who referred the applicant from the drop-down menu.

Job section
^^^^^^^^^^^

> The following fields should be pre-populated when creating a new applicant. Altering of these fields

Author: If they were entered on the job position, they appear. Department can be blank, so if it is, it won't populate. I added clarification.

- :guilabel:`Expected Salary`: Enter the amount the applicant is requesting for the role in this
  field. The number should be in a `XX,XXX.XX` format.
- :guilabel:`Expected Salary - Extra advantages...`: If any extra advantages are requested by the
  applicant, enter it in this field. This should be short and descriptive, such as `1 week extra
  vacation` or `dental plan`.
- :guilabel:`Proposed Salary`: Enter the amount to be offered to the applicant for the role in this
  field. The number should be in a `XX,XXX.XX` format.
- :guilabel:`Proposed Salary - Extra advantages...`: If any extra advantages are beng offered to the
  applicant, enter it in this field. This should be short and descriptive, such as `1 week extra
  vacation` or `dental plan`.
- :guilabel:`Availability`: Using the calendar module in the drop-down menu, select the available
  start date for the applicant.

Application summary tab
^^^^^^^^^^^^^^^^^^^^^^^

Any additional details or notes that should be added to the applicant's card can be typed into this
field.

.. image:: recruitment/new-applicant.png
   :align: center
   :alt: All the fields for a new applicant form entered.

First interview
---------------

> After an applicant has passed the :guilabel:`Initial Qualification` stage, they next move to the

Reviewer 2: ```suggestion
After an applicant has passed the :guilabel:`Initial Qualification` stage, they can be moved to the
```

:guilabel:`First Interview` stage. To move the applicant to the next stage, drag and drop the
applicant's card to the :guilabel:`First Interview` stage, or click on the :guilabel:`First
Interview` stage button at the top of the individual applicant's card.

When the applicant's card moves to the :guilabel:`First Interview` stage, an acknowledgement email
is automatically sent to the applicant. The email is created using an email template titled
:guilabel:`Applicant: Acknowledgement`. This email template is pre-configured in Odoo, and contains
dynamic placeholders for the job position, the recruiter's name, and the company's information. Once
the email is sent, the email appears in the chatter.

> .. image:: recruitment/applicant-acknowledgement.png

Author: It is, but it's less to read and more to see where it is stored, and to highlight where to look.

         email is seen in the chatter, and is sent using a template.

.. _recruitment/schedule:

Schedule a meeting
~~~~~~~~~~~~~~~~~~

When a date and time have been agreed upon for both the applicant and interviewer, the interview can
be scheduled. To schedule an interview, whether a phone or in-person interview, click the
:guilabel:`Meeting` smart button at the top of the applicant's record.

.. note::
   The meeting smart button will display :guilabel:`No Meeting` if no meetings are currently
   scheduled. For new applicants who are new to the :guilabel:`First Interview` stage, this will be
   the default. If there are any meetings already scheduled, the smart button will display
   :guilabel:`Next Meeting` with the date of the upcoming meeting.

The calendar view loads, showing the currently scheduled meetings and events. The meetings and
events displayed are for the employees who are listed under the :guilabel:`Attendees` section on the
right side of the calendar view. To change the currently loaded meetings and events being displayed,
uncheck the person who's calendar events are to be hidden. Only the checked attendees will be

> visible on the calendar.

Author: No- BUT there should be no reason someone unassociated with recruitment/interviews/etc is creating a meeting in the DB - at east that's what I was thinking. Id never go in and schedule an interview for our recruitment team, you know? So you can add people to the meeting, I just honestly can't imagine a scenario where another person was scheduling this- since recruitment is typically more confidential. You know? I am not sure if that answers your comment fully but I think it does. I didn't add the search since that seemed to be... how should I say this- common knowledge on how it works since it's all over the DB? I always struggle with just how granular to go- because I end up with docs this long  ^_^ But in the end I thought this was an understood chunk.

The default view is the week view. To change the calendar view, click on a smart button to present
the calendar in a different view, either :guilabel:`Day`, :guilabel:`Month`, or :guilabel:`Year`.
To change the displayed date range for the calendar, either use the :guilabel:`⬅️ (Left)`,
:guilabel:`➡️ (Right)`, or :guilabel:`Today` smart buttons above the calendar, or click on a date in

> the calendar module on the right side of the displayed calendar.

Author: These steps are sill correct though- and appear when scheduling a meeting.

.. image:: recruitment/calendar.png
   :align: center
   :alt: The calendar view, highlighting how to change the displayed meetings.

> 

Author: I sized it to contain all the info I wanted to show- which is the buttons on the far left and the calendar view on the far right. This was the best I could do to get everything without cropping in a weird way.

  adjusts to the correct duration length.
- :guilabel:`All Day`: If the meeting is an all day long event, check the box next to :guilabel:`All
  Day`. After this box is checked, the :guilabel:`Duration` field is hidden from view, a well as the
  meeting start and end times.
- :guilabel:`Organizer`: The employee who created the meeting is populated in this field. Use the
  drop-down menu to change the selected employee.
- :guilabel:`Description`: Enter a brief description in this field. There is an option to enter
  formatted text, such as numbered lists, headings, tables, as well as links, photos, and more. Type
  `/` and a list of options are presented. Scroll through the options and click on the desired item.
  The item appears in the field and can be modified. Each command presents a different pop-up
  window. Follow the instructions for each command to complete the entry.
- :guilabel:`Reminders`: Select a reminder from the drop-down menu. Options include
  :guilabel:`Notification`, :guilabel:`Email`, and :guilabel:`SMS Text Message`, each with a
  specific time period before the event (hours, days, etc). The reminder chosen will alert the

>   meeting participants of the meeting via the selected option.

Author: Yes, that is how I explained it- there are notifications, emails, or texts- each with a timed duration. So you see Notifications fora variety of times- this is correct.

- :guilabel:`Location`: Enter the location for the meeting.
- :guilabel:`Meeting URL`: If the meeting is virtual, enter the meeting link in this field.
- :guilabel:`Tags`: Select any tag(s) for the meeting using the drop-down menu. There is no limit to
  the number of tags that can be used.

Options tab
^^^^^^^^^^^

- :guilabel:`Recurrent`: If the meeting should repeat at a selected interval (not typical for a
  first interview), check the box next to :guilabel:`Recurrent`. This reveals a recurrence module.

>   Enter the details for how often the meeting should repeat, including an end time.

Reviewer 2: this isn't a module

Author: I changed it to 'section'

- :guilabel:`Privacy`: Select the visibility of this meeting. Options are :guilabel:`Public`,
  :guilabel:`Private`, and :guilabel:`Only internal users`. :guilabel:`Public` allows for everyone
  to see the meeting, :guilabel:`Private` allows only the attendees listed on the meeting to see the

>   meeting, and :guilabel:`Only internal users` allows everyone in the company to see the meeting.

Reviewer 2: Portal users? I'm not sure where to view this... I don't think we have this posted on the website... I may need to reach out to a PO on this one.... unless you know the permissions.

Author: I had checked with Quentin on this, and I edited this so its the 'logged into the company's DB' which I think sounds better. If you are logged into the system, you can see it.

- :guilabel:`Show as`: Select from the drop-down whether the meeting time should appear as
  :guilabel:`Busy` or :guilabel:`Available` in the attendees' schedules. Setting this field to
  :guilabel:`Available` allows other meetings to be scheduled for the same time. It is recommended
  to set this field to :guilabel:`Busy`, so no other meetings can be scheduled for the attendees at
  the same time.

.. image:: recruitment/new-event.png
   :align: center
   :alt: A new meeting card with all the details populated, and ready to save.

Send meeting to attendees
*************************

Once changes have been entered and the meeting details are correct, the meeting can be sent to the
attendees via email or text message.

To send the meeting via email, click the :guilabel:`Email` button next to the list of attendees. A

> :guilabel:`Contact Attendees` pop-up appears. A pre-formatted email using the default

Reviewer 2: ```suggestion
:guilabel:`Contact Attendees` email configurator pop-up appears. A pre-formatted email using the default
```

> :guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
> the document (job application), as well as the :guilabel:`Recruiter` are added as
> :guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email

Reviewer 2: The organzer is not added on as a attendee and it doesn't add them onto their schedule. 
```suggestion
:guilabel:`Calendar: Event Update` email template populates the email body field. The followers of
the document (job application), as well as the :guilabel:`Recruiter` are added as
:guilabel:`Recipients` by default. Add the applicant's email address to the list to send the email
```
The above statement isn't true

Author: I just tested this and rechecked everything a couple times. I don't mention the organizer, so I don't know what part of this you are referring to. I did retry this and you are correct- I was logged in as the recruiter, so it confused me, I thought it was adding the recruiter- it's just the followers and whoever is making the meeting,

selected. The second line lists the number of recipients and how many of them are invalid. If a
contact does not have a valid phone number listed, click :guilabel:`Close`, and edit the attendee's
record, then redo these steps.

.. warning::
   It is *not* recommended to check the box to send to all the records in the :guilabel:`Send SMS
   Text Message` pop-up. This would send a message to all contacts with a valid phone number.

When the message reads :guilabel:`0 are invalid`, type in the message to be sent in the
:guilabel:`Message` field. Beneath the message field, the number of characters, as well as the
amount of text messages required to send the message (according to GSM7 criteria) appears. Click
:guilabel:`Put In Queue` to have the text sent later, after any other messages are scheduled, or
click :guilabel:`Send Now` to send the message immediately.

.. image:: recruitment/send-sms.png
   :align: center
   :alt: Send a text message to the attendees of the meeting.

.. note::
   Sending text messages is not a default capability with Odoo. To send text messages, credits are
   required, which need to be purchased. For more information on credits and plans, refer to

>    :doc:`this documentation <../marketing/sms_marketing/pricing/pricing_and_faq>`.

Reviewer 2: IAP credits -- note

Reviewer 2: reference docs: https://www.odoo.com/documentation/16.0/applications/general/in_app_purchase.html

Author: Thank you! I thought the other was the right doc- I was wrong =)!

Second interview
----------------

> After an applicant has passed the :guilabel:`First Interview` stage, they next move to the

Reviewer 2: ```suggestion
After an applicant has passed the :guilabel:`First Interview` stage, they can be moved to the
```

:guilabel:`Second Interview` stage. To move the applicant to the next stage, drag and drop the
applicant's card to the :guilabel:`Second Interview` stage, or click on the :guilabel:`Second
Interview` stage button at the top of the individual applicant's card.

When the applicant's card moves to the :guilabel:`Second Interview` stage, there are no automatic
activities or emails configured for this stage, unlike for the :guilabel:`First Interview` stage.
The recruiter should :ref:`schedule a second interview <recruitment/schedule>` with the applicant,

> following the same process as for the first interview.

Reviewer 2: It sounds like you're telling the customer how to do their recruitment process- may instead of saying should, use can

Reviewer 2: We offer the software to accommodate their recruitment processes-- not tell them how.

Author: I am telling them how the default recruitment flow is configured in Odoo and how the flow works. So if they downloaded the Recruitment app and started to recruit for a position out of the box, I wrote this for them to follow the steps created, so they understand the flow. I know they can modify it, but I think they should know how t's configured so they can make the changes being fully informed. Example- if there are automatic emails sent for a stage and they just rename it because they don't know how it works, etc- that's a problem.  I covered this in the opening paragraph, that this is how it's done out of the box but can be modified. I will try to do the 'can' language, but that's why I wrote it the way I did.

Contract proposal
-----------------

> When the applicant has passed the interview stages and an offer is ready to be sent, they move to
> the :guilabel:`Contract Proposal` stage. Drag and drop the applicant's card to the
> :guilabel:`Contract Proposal` stage, or click on the :guilabel:`Contract Proposal` stage button at
> the top of the individual applicant's card.

Reviewer 2: not all applicants to through each stage--- they can be moved.
```suggestion
When the applicant has passed the interview stages and an offer is ready to be sent, they can be moved to
the :guilabel:`Contract Proposal` stage. Drag and drop the applicant's card to the
:guilabel:`Contract Proposal` stage, or click on the :guilabel:`Contract Proposal` stage button at
the top right of the individual applicant's card.
```

The next step is to send an offer to the applicant. On the applicant's card, click the
:guilabel:`Generate Offer` smart button. A :guilabel:`Generate a Simulation Link` pop-up appears
All fields are pre-populated with information from the job position, except for the :guilabel:`Force
New Cars List`, :guilabel:`Default Vehicle`, and the :guilabel:`Canteen Cost` (this is set to
`0.00`).

.. note::
   Not all fields may appear on the :guilabel:`Generate a Simulation Link` pop-up. Depending on the
   localization setting for the company and the applications installed, some fields may not appear.
   For example if the *Fleet* application is not installed, the fields related to vehicles will not
   appear on the pop-up.

Universal fields
~~~~~~~~~~~~~~~~

The following fields appear for all offers sent to applicants regardless of localization settings.

> - :guilabel:`Offer Template`: The template currently being used to populate the :guilabel:`Generate

Reviewer 2: The offer template can also be accessed by going to configuration--> JOb Positions --> select the position--> recruitment tab--> Contract Template

Reviewer 2: It's likely edits will be made manually and not on the fly with this document

Author: I go into depth in the other doc, so I didn't add any instructions that overlapped with making these form scratch.

  a Simulation Link` pop-up. To modify the template, click the :guilabel:`External link` icon to the
  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
- :guilabel:`Job Title`: The name of the position being offered to the applicant.
- :guilabel:`Contract Start Date`: The date the contract takes effect. The default date is the

>   current date. To modify the date, click the drop-down to reveal the calendar module. Use the

Reviewer 2: not the calendar module

Reviewer 2: this is simply a date selector

  :guilabel:`< (left)` and :guilabel:`> (right)` arrows on either side of the month to navigate to
  the desired month, then click on the :guilabel:`day` to select the date.
- :guilabel:`Yearly Cost`: The annual salary being offered.
- :guilabel:`Offer Link`: This link takes the applicant to a webpage where they can enter their
  personal information, which will be imported to their employee record when created. If applicable,
  the applicant can modify their salary package (this option is not available for all
  localizations). Lastly, this is where the applicant accepts the offer by clicking

>   :guilabel:`Review Contract & Sign` to accept the contract and sign it.

Reviewer 2: using the sign application

Optional fields
~~~~~~~~~~~~~~~

> The following fields will only appear if specific localization settings are selected, and/or certain

Reviewer 2: which applications prompt these options?

applications are installed.

- :guilabel:`Job Position`: The official title of the job being offered to the applicant, from the
  :guilabel:`Job Position` card. To modify the name, click the :guilabel:`External link` icon to the
  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
- :guilabel:`Department`: The department the job position falls under. To modify the department,
  click the :guilabel:`External link` icon to the right of the drop-down arrow. Make any changes,
  then click :guilabel:`Save`.
- :guilabel:`Force New Cars List`: Check this box to offer a new vehicle to the applicant.
- :guilabel:`Default Vehicle`: Using the drop-down menu, select the vehicle that will be assigned
  to the applicant.
- :guilabel:`Canteen Cost`: Enter the amount that is offered to the applicant for the cafeteria.
- :guilabel:`Contract Type`: Using the drop-down menu, select the type of contract that is being
  offered.

.. image:: recruitment/generate-link.png
   :align: center

>    :alt: Format the offer to send to the applicant. Make any modifications before sending.

Reviewer 2: I would like to see some annotations here --

Author: I disagree- i would have to highlight every single entry since every entry is explained in the bulleted lists. There is no  single piece of info that is more important than another, and I think they can easily follow the explanations with all the fields, they are clearly labeled, etc.

Send offer
~~~~~~~~~~

To send the offer to the applicant, click :guilabel:`Send`. An email pop-up appears, using the
:guilabel:`Applicant: Your Salary Package` email template. The :guilabel:`Recipients`,

> :guilabel:`Subject`, and email body are pre-populated based on the email template. If any

Reviewer 2: A save contact option pop-up happened first!

Author: True- if they don't have an email on file. I will add that!

attachments need to be added, click the :guilabel:`Attachments` button and a file explorer window
appears. Navigate to the desired file, then click :guilabel:`Open` to attach it to the email. The
attachment loads, and is listed above the :guilabel:`Attachments` button. Once the email is ready to
send, click :guilabel:`Send`.

.. image:: recruitment/send-offer.png
   :align: center
   :alt: Send an email to the applicant with a link to the offered salary.

Contract signed
---------------

Once the applicant has accepted the offer and signed the contract, the next step is to move the
applicant to the :guilabel:`Contract Signed` stage. To move the applicant to the next stage, drag

> and drop the applicant's card to the :guilabel:`Contract Signed` stage, or click on the

Reviewer 2: this stage is hidden in the drop-down arrow

menu. The default refuse reasons in Odoo are: :guilabel:`Doesn't fit the job requirements`,
:guilabel:`The applicant is not interested anymore`, and :guilabel:`The applicant gets a better
offer`. These three refusal reasons have pre-configured email templates associated with them.
Additional refusal reasons can be created and existing ones can be modified or deleted. To add a new
refusal reason, type in the name of the reason, then click :guilabel:`Create and Edit...`. A
:guilabel:`Create: Refuse Reason` pop-up window appears. Type in the name of the new email template
in the :guilabel:`Email Template` field, and click :guilabel:`Create and Edit...`. A
:guilabel:`Create: Email Template` window loads. Configure the new email template, then click
:guilabel:`Save`.

Email templates in Odoo can use dynamic placeholders, which are customized pieces of data that
populate dynamic content. For example, the applicant's name can be a piece of dynamic content. The
applicant's name will appear anytime that dynamic placeholder appears on the email template. For
more detailed information on email templates, refer to :doc:`this documentation

> <../general/email_communication/email_template>`.

Author: Funny you mention the manual placeholder- I didn't think I was implying they could change anything, just explaining what a dynamic placeholder was, and how they are used, with an example to really get the point across. I'll explain that they are not manual changes in 15. I believe I originally only added this info because it was suggested  to explain wat exactly a dynamic placeholder was.

> <../general/email_communication/email_template>`.

Author: I mentioned these are for "pre-configured" email templates and they MIGHT/MAY contain dynamic placeholders- then I explain what that is. i think that's more clear.

  right of the drop-down arrow. Make any changes, then click :guilabel:`Save`.
- :guilabel:`Job Title`: The name of the position being offered to the applicant.
- :guilabel:`Contract Start Date`: The date the contract takes effect. The default date is the
  current date. To modify the date, click the drop-down to reveal a calendar. Use the :guilabel:`<
  (left)` and :guilabel:`> (right)` arrows on either side of the month to navigate to the desired
  month, then click on the :guilabel:`day` to select the date.
- :guilabel:`Yearly Cost`: The annual salary being offered.
- :guilabel:`Offer Link`: This link takes the applicant to a webpage where they can enter their
  personal information, which will be imported to their employee record when created. If applicable,
  the applicant can modify their salary package (this option is not available for all
  localizations). Lastly, this is where the applicant accepts the offer by clicking
  :guilabel:`Review Contract & Sign` to accept the contract and sign it using the *Sign*
  application.

.. note::
   The *Sign* application must be installed in order for an applicant to sign any document,

>    including a contract. Ensure the *Sign* application is installed to use this feature.

Reviewer 2: This sounds like the applicant needs the sign app

Author: I changed the wording so it doesn't say "applicant" and implies only the user needs the sign app (which is correct).

Send offer
~~~~~~~~~~

To send the offer to the applicant, click :guilabel:`Send`. If the applicant does not have an email
address listed on their applicant card, the applicant form appears so their email can be configured.
If there is already an email configured on the applicant's card, the email pop-up appears instead.
If prompted with the applicant's card, enter the applicant's email address in the :guilabel:`Email`
field, then click :guilabel:`Save`. The applicant form closes and the email pop-up appears.

The :guilabel:`Applicant: Your Salary Package` email template is used, and the
:guilabel:`Recipients`, :guilabel:`Subject`, and email body are pre-populated based on the email
template. If any attachments need to be added, click the :guilabel:`Attachments` button and a file
explorer window appears. Navigate to the desired file, then click :guilabel:`Open` to attach it to
the email. The attachment loads, and is listed above the :guilabel:`Attachments` button. Once the
email is ready to send, click :guilabel:`Send`.

.. note::
   To send an offer, ensure the *Sign* application is installed. This is necessary so the offer can

>    be accepted and signed by the applicant.

Reviewer 2: I would specify that the applicant doesn't need the app

.. image:: recruitment/send-offer.png
   :align: center
   :alt: Send an email to the applicant with a link to the offered salary.

Contract signed
---------------

Once the applicant has accepted the offer and signed the contract, the next step is to move the
applicant to the :guilabel:`Contract Signed` stage. To move the applicant to the next stage, drag
and drop the applicant's card to the :guilabel:`Contract Signed` stage, or click the
:guilabel:`More` button at the top of the applicant's card to reveal the :guilabel:`Contract

> Signed` stage button at the top of the individual applicant's card, nd click :guilabel:`Contract

Reviewer 2: ```suggestion
Signed` stage button at the top of the individual applicant's card, and click :guilabel:`Contract
```

I'm noticing errors in the changes you're making...

in gray and the applicants in it are hidden from view. To expand the folded stage and view the
applicant cards for that column, click anywhere on the thin gray column that says the stage name and
the column expands, revealing the applicants.

.. image:: recruitment/stages.png
   :align: center
   :alt: Expand a folded column by clicking on it in the kanban view.

Each stage has a color-coded bar beneath the stage name, giving information about the applicant's
status in the stage. The colors are:

- :guilabel:`Green`: There are applicants in the stage with an activity that is either already
  scheduled or needs to be scheduled in the future (such as a phone call or interview), according to
  the parameters for that particular stage.
- :guilabel:`Yellow`: There are applicants in the stage with an activity either scheduled for today
  or due to be scheduled today.
- :guilabel:`Red`: There are applicants in the stage with overdue activities which need to be
  scheduled.
- :guilabel:`Gray`: There are either no activities currently scheduled and no future activities that
  are required to be scheduled, or there are no applicants currently in the stage.

> .. _stages:

Author: I didn't realize there was a naming convention for the anchors- I was just so thrilled when I was able to do them, and in this one I went HAM! I have been using them more and more, since I really like chopping up the sections, and LOVE it if I (or a reader) can just to a corresponding spot. I adore anchors =) 

I will be sure to do very descriptive and appropriate anchors from now on. LMK if there is a limit, or something else I might be doing wrong by using so many.

Customize stages
================

Stages can be modified, added, or deleted to best meet the needs of the particular hiring steps of
a business.

New stage
---------

To create a new stage, click on :guilabel:`+ Add a Column` and a new column appears. Enter the title
for the new stage in the :guilabel:`Column title` field, then click :guilabel:`Add`. The new column
appears, and another new stage is available to create. If no new stages are needed, click anywhere
on the screen to exit the new column creation.

.. image:: recruitment/add-column.png
   :align: center

>    :alt: The gear icon that appears when a column name is moused over, and the menu it displays when

Reviewer 3: This alt text does not seem to apply to this image in context, I suppose it is for the next image that is missing an alt tag description 😄

         clicked.

.. _modify:

Modify stage
------------

To modify the settings of a stage, hover over the name of the stage, and a :guilabel:`⚙️ (gear)`
icon appears in the upper right hand side of the stage. Click on the :guilabel:`⚙️ (gear)` icon and
a menu appears. Then click on the :guilabel:`Edit Stage` option. An :guilabel:`Edit Column` form
appears. Make any modifications to the form, then click :guilabel:`Save` when done.

.. image:: recruitment/gear.png
   :align: center

>    :alt: alt text

Reviewer 3: Missing alt text description

  If this is active, a :guilabel:`Points` field appears. Enter the amount of referral points
  the employee receives when an applicant enters this stage.
- :guilabel:`Tooltips` section: There are three pre-configured labels (colored circles) for each
  applicant's card, indicating its status. These colors are displayed at the top of each stage to
  reflect the statuses of the applicants in the stage. The *names* for the label can be modified,
  but the label itself (the color) cannot. The default names and labels are: :guilabel:`In Progress`
  (gray), :guilabel:`Blocked` (red), and :guilabel:`Ready for Next Stage` (green).
- :guilabel:`Requirements`: Enter any internal notes for this stage explaining any requirements of
  the stage.

Delete stage
------------

If a stage is no longer needed, the stage can be deleted. To delete a stage, hover over the name of
the stage, and a :guilabel:`⚙️ (gear)` icon appears. First, click on the :guilabel:`⚙️ (gear)` icon
to reveal a drop-down menu, then click :guilabel:`Delete`. A pop-up warning appears, asking
:guilabel:`Are you sure you want to remove this column?`. Click :guilabel:`OK` to delete the column.

> .. Important::

Reviewer 3: Admonition tags, and most (I think all?) RST directives should be lowercase
```suggestion
.. important::
```

  passed the :guilabel:`Initial Qualification` stage and they will be contacted to set up an
  interview with the recruiter.
- :guilabel:`Applicant: Not interested anymore`: This template is used when an applicant
  communicates that they are no longer interested in the position, and thanks them for their time
  and consideration.
- :guilabel:`Applicant: Refuse`: This template is used when an applicant is no longer being
  considered for the position.
- :guilabel:`Applicant: Your Salary Package`: This template is used when offering an applicant a
  position. This informs the applicant they can configure their salary package.

.. note::
   Email templates can be created, modified, and deleted to suit the needs of a business. For more
   information on email templates, refer to the :doc:`../general/email_communication/email_template`
   document.

To manually send an email click :guilabel:`Send message` in the chatter. A text box appears, as well
as the applicant's email address. Click the :guilabel:`⤢ (double arrow)` full composer icon in the
bottom right corner of the :guilabel:`Send Message` tab in the chatter.

.. image:: recruitment/full-composer.png
   :align: center

>    :alt: Send an email using a template by clicking the full composer icon in the bottom right of
>          the send message tab in the chatter.

Reviewer 3: No need for the alt tag to repeat the previous instruction, it can be simplified to only describe the image:
```suggestion
   :alt: Send an email from the chatter.
```

   To send an offer, ensure the *Sign* application is installed. This is necessary so the offer can
   be sent to the applicant by the recruiter. The applicant does not need any software installed.

.. image:: recruitment/send-offer.png
   :align: center
   :alt: Send an email to the applicant with a link to the offered salary.

.. _signed:

Contract signed
---------------

Once the applicant has accepted the offer and signed the contract, the next step is to move the
applicant to the :guilabel:`Contract Signed` stage. To move the applicant to the next stage, drag
and drop the applicant's card to the :guilabel:`Contract Signed` stage, or click the
:guilabel:`More` button at the top of the applicant's card to reveal the :guilabel:`Contract
Signed` stage button at the top of the individual applicant's card, and click :guilabel:`Contract
Signed`.

The :guilabel:`Contract Signed` stage is folded in the kanban view, but the card may still be
dragged and dropped into that stage.

Once the applicant's card moves to the :guilabel:`Contract Signed` stage, a green :guilabel:`Hired`
banner appears in the top right of the applicant's card.

.. image:: recruitment/hired.png
   :align: center

>    :alt: A large green "HIRED" banner appears in the top right corner of an applicant's card after
>          they are moved ot the Contract Signed stage.

Reviewer 3: Typo and no need for the alt tag to restate the previous paragraph- it can just describe the image :)
```suggestion
   :alt: Hired banner in the top right corner of applicant card.
```
